mcm/tests/campaign/test_campaign.py

```diff
@@ -0,0 +1,266 @@
+"""
+This module tests the API operations related with
+the campaign entity.
+"""
+
+import uuid
+from pathlib import Path
+from copy import deepcopy
+import pytest
+from tests.base_test_tools import Entity, EntityTest, api
+from tests.api_tools import McM, Roles
+
+
+class CampaignAPI(Entity):
+    """
+    Implements a simple test API client focused
+    only on the campaign operations.
+    """
+
+    def __init__(self, mcm: McM) -> None:
+        mock_path = Path("./tests/campaign/static/campaign.json").absolute()
```

Comment:
Use `resource`

Reply:
I am not sure about what you mean, please provide an example

Reply:
```py
from importlib.resources import files
import campaign
mock_data = json.loads(files(campaign).joinpath("static/campaign.json").read_text())
```

Reply:
Nice, I didn't know this. Thanks for the explanation. I am going to perform the change